pandas/core/series.py

```diff
@@ -3046,21 +3046,21 @@ def _gotitem(self, key, ndim, subset=None):
     Examples
     --------
 
-    >>> s = pd.Series(np.random.randn(10))
+    >>> s = pd.Series(range(1,10,1))
```

Comment:
As it doesn't make a difference for the example, I'd use `[1, 2, 3, 4]` instead of range (it makes the example easier to understand). In any case, please check pep8 in the examples, there are missing spaces after the commas.

Reply:
Can you also show `>>> s` after creating it, and remove the blank line between the examples title and the code.